apps/namadillo/src/integrations/Keplr.ts

```diff
@@ -8,21 +9,55 @@ import { WalletConnector } from "./types";
 import { basicConvertToKeplrChain } from "./utils";
 
 type Signer = OfflineAminoSigner & OfflineDirectSigner;
-const keplr = (window as KeplrWindow).keplr!;
 
 export class KeplrWalletManager implements WalletConnector {
+  install(): void {
+    console.warn("Keplr is not available. Redirecting to the Keplr download page...");
+    window.open("https://www.keplr.app/get", "_blank");
+  }
```

Comment:
I think this is good! Though we may want to invoke `install()` from outside of the integration (for example, if `get()` doesn't return Keplr, we could display a `Download` button, which in turn invokes `install()`, instead of calling it automatically). I wanted to see what @pedrorezende thought of this too, as we've been chatting about how to handle that scenario for Keplr. Perhaps we can keep this method (`install()`), but invoke it outside of `get()`?

Reply:
That's an idea indeed! So commenting out the part where it automatically calls install, but keep the install method so it's callable? 

It would also be an idea to have like an `isInstalled()` method to quickly see whether it is around or not and base the button on this boolean value.

I'm not behind the pc atm, but I could remove the part where it auto calls install inside of the get!

Reply:
Yeah, I think for now, just take out `if(!keplr) this.install();` so that it's not automatically called. We need to figure out exactly how to handle the Download button still on our end, and we can certainly address that in another PR!

Reply:
I agree with @jurevans , later today I can hook this `install` method somewhere in order to download Keplr after clicking on a button to do so. Thank you very much for this PR @zenodeapp !